README.md

```diff
@@ -57,7 +57,7 @@ Options:
 A Standalone image can be created by running the command below in the directory where the solution is located.
 
 ```
-$ docker build . -t <IMAGE_TAG>
+$ docker build . -t <IMAGE_TAG> --build-arg COMMIT=<VERSION_SUFFIX>
```

Comment:
I guess it would be better if adding mention about `VERSION_SUFFIX`.

Reply:
Should I mention that we use VERSION_SUFFIX for committing our latest git commit on our docker hub repo but that this could be any value when users build from their local?

Reply:
Right 👍